src/lib/test/test_ecr_repos.py

```diff
@@ -0,0 +1,48 @@
+from lib.ecr_repos import ECRRepo, ECRRepos
+
+from lib.test.fixtures.ecr_interface import (
+  mock_ecr_client, # indirect
+  mock_ecr_describe_repositories_result,
+  mock_ecr_interface,
+  mock_ecr_list_tags_for_resource_result
+)
+
+def describe_ecr_repo():
```

Comment:
Any reason why you're not extending `TestCase` for these as per the pytest [doc](https://docs.pytest.org/en/7.1.x/how-to/unittest.html)? How are you running these tests?

Reply:
> Any reason why you're not extending TestCase...

I thought that with Pytest, it's no longer required. That doc is about using Pytest for legacy unittest-based test suites?

Reply:
> How are you running these tests?

`make test` (run all tests in repo)

`pytest src/lib/` (run tests in src/lib dir)

Reply:
Ah I see... you're relying on filenaming conventions for picking up test definitions? I'm just wondering how test discovery works.

Reply:
yes, Pytest [looks for](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery) `test` in dir names and function names.

then there's the [pytest-describe lib](https://pypi.org/project/pytest-describe/) which i think parses the `describe` in function names.